data/pools/transactionPool.go

```diff
@@ -295,15 +304,15 @@ func (pool *TransactionPool) Remember(txgroup []transactions.SignedTxn) error {
 	pool.mu.Lock()
 	defer pool.mu.Unlock()
 
+	if pool.pendingBlockEvaluator == nil {
+		return fmt.Errorf("TransactionPool.Remember: no pending block evaluator")
+	}
+
```

Comment:
this wouldn't hurt, but it's a duplicate. it's the first thing being done in pool.test

Reply:
I just moved a block of code a few lines. Maybe we should delete it.